components/FruitBasket.js

```diff
@@ -0,0 +1,32 @@
+const React = require('react');
+const { Component } = require('react');
```

Comment:
We can remove this import, `React` is imported above and `Component` isn't used in this file